### DISCUSSION TITLE
Top navbar hover

### DISCUSSION THREAD

**Discussion by Lalremruata** [filamentphp/filament#8967] [Help]
Hi, i want the top navbar list to show on hover, is there a way to do it.

**Comment by leandrocfe** [comment]
1 - [Create a custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)
2 - Customize the css using the theme.css file

```css
.fi-topbar-item a:hover {
    @apply text-primary-600;
}
```

**Reply by leandrocfe** [reply]
Did you [try it](https://discordapp.com/channels/883083792112300104/1160858912568901663/1160948528806371328)?
Close or mark as answered if it worked, please
